tests/e2e/test_raster_analysis.py

```diff
@@ -115,48 +114,43 @@ def test_extent_2010(context):
     result = tiled_handler({"geometry": IDN_24_9_GEOM, "query": query}, context)["body"]
     assert result["status"] == "success"
 
-    record_results = pd.read_csv(StringIO(result["data"])).to_dict(orient="records")
-    assert record_results[0]["area__ha"] == pytest.approx(
+    assert result["data"][0]["area__ha"] == pytest.approx(
         IDN_24_9_2010_EXTENT["area__ha"], 0.01  # TODO slightly more off than expected
     )
 
 
 def test_lat_lon(context):
-    query = "select latitude, longitude, umd_glad_alerts__date from data where umd_glad_alerts__date > '2020-01-01'"
-    result = tiled_handler({"geometry": IDN_24_9_GEOM, "query": query}, context)["body"]
+    query = "select latitude, longitude, umd_glad_alerts__date from data where umd_glad_alerts__date >= '2019-01-01' and umd_glad_alerts__date < '2020-01-01'"
+    result = tiled_handler(
+        {"geometry": IDN_24_9_GEOM, "query": query, "format": "csv"}, context
+    )["body"]
     assert result["status"] == "success"
 
-    record_results = pd.read_csv(StringIO(result["data"])).to_dict(orient="records")
-    assert record_results
+    lines = result["data"].splitlines()
+    assert len(lines) == 317920
```

Comment:
Hmm, where is this number coming from? Could you put it in a constant with a nice name or comment?

Reply:
Oops, sorry, forgot  this was in a test, not regular code. Ignore previous comment. :)

Reply:
You're right, I should still put it in fixtures file even though it's just one value